pkg/crc/cluster/cluster.go

```diff
@@ -179,16 +179,43 @@ func EnsureGeneratedClientCAPresentInTheCluster(ocConfig oc.Config, sshRunner *s
 }
 
 func RemovePullSecretFromCluster(ocConfig oc.Config, sshRunner *ssh.Runner) error {
+	mcBeforeRemovePullSecret, stderr, err := ocConfig.RunOcCommand("get mc --sort-by=.metadata.creationTimestamp --no-headers -oname")
+	if err != nil {
+		return fmt.Errorf("failed to get machineconfig resource %w: %s", err, stderr)
+	}
 	logging.Info("Removing user's pull secret from instance disk and from cluster secret...")
 	cmdArgs := []string{"patch", "secret", "pull-secret", "-p",
 		`'{"data":{".dockerconfigjson":"e30K"}}'`,
 		"-n", "openshift-config", "--type", "merge"}
 
-	_, stderr, err := ocConfig.RunOcCommand(cmdArgs...)
+	_, stderr, err = ocConfig.RunOcCommand(cmdArgs...)
 	if err != nil {
 		return fmt.Errorf("Failed to remove Pull secret %w: %s", err, stderr)
 	}
 
+	mcRenderedFunc := func() error {
+		mcAfterRemovePullSecret, stderr, err := ocConfig.RunOcCommand("get mc --sort-by=.metadata.creationTimestamp --no-headers -oname")
+		if err != nil {
+			return &errors.RetriableError{Err: fmt.Errorf("failed to get machineconfig resource %w: %s", err, stderr)}
+		}
+		if mcBeforeRemovePullSecret == mcAfterRemovePullSecret {
+			return &errors.RetriableError{Err: fmt.Errorf("machine config is not rentered after removing pull secret")}
```

Comment:
small typo `rentered` should be `rendered`

Reply:
Fixed